src/app/components/feed/FeedImportDialog.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import Relay from 'react-relay/classic';
 import PropTypes from 'prop-types';
-import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
+import { FormattedMessage, injectIntl } from 'react-intl';
```

Comment:
can we also remove `injectIntl` ?

Reply:
seems like it, the build didn't pester me about removing it so I missed it! Thanks